backend/cybnetics/endpoints.py

```diff
@@ -1,8 +1,118 @@
-from flask import Flask, jsonify
+import json
+import ast
+import jwt
+import datetime
+
+from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
-@app.route('/test', methods=['GET'])
-def test():
-    return jsonify({'hello': 'world'})
+
+db = app.config['DATABASE']
```

Comment:
i added some code in the other pr that basically does something like this but in a before_first_request hook so you can remove this line